src/dso_api/dynamic_api/remote/serializers.py

```diff
@@ -43,7 +43,7 @@ class RemoteListSerializer(DSOListSerializer):
 
     @property
     def expanded_fields(self):
```

Comment:
Maybe add type annotation. Makes it easier to read the code.

Reply:
tx! :-)